package.json

```diff
@@ -12,6 +12,7 @@
     "arbify": "yarn ts-node src/main.ts --action arbify",
     "permit": "yarn ts-node src/main.ts --action permit",
     "fullList": "yarn ts-node src/main.ts --action full --tokenList full",
+    "allTokensList": "yarn ts-node src/main.ts --action alltokenslist --tokenList full",
```

Comment:
Tthe task requires you specify an `--l2NetworkID`, like the other tasks.
It's only _recommended_ for testnet since generating a full, nicely formatted token list for other networks is likely a bad idea (scams, etc).

Do you think that's sufficient (/ does that answer your question or am I misunderstanding?)

Reply:
NIT: I'd name it with a verb. The same way we have `update`, 'arbify', 'serve', etc